### PR TITLE
web: ops pages on mock data — Machines, Machine, Pair, Schedules, Plugins, Settings, History, Usage

### LINKED ISSUE #90: web: ops pages on mock data — Machines, Machine, Pair, Schedules, Plugins, Settings, History, Usage
## Goal

The operations routes — Machines `/machines`, Machine `/machines/:id`, Pair `/pair`, Schedules `/schedules`, Plugins `/plugins`, Settings `/settings`, and the new History `/history` and Usage `/usage` — render the handoff's structure on mock data, ready for #36 / #42 / #48 / #44 / #45 to wire actors.

## Requirements

EXE-02, EXE-06, EXE-07, EXE-12, USR-04, AST-02..AST-07, PLG-09, OPS-03, OPS-07, OPS-10 — see `docs/requirements.md`.

## Design source

`docs/design/HANDOFF.md` → "Screen specs" rows Machines through Usage; "Content grids"; table column templates under "Layout and shell"; screenshots `Machines.png`, `Machine.png`, `Pair.png`, `Schedules.png`, `Plugins.png`, `Settings.png`, `History.png`, `Usage.png` and their artboards.

## Owner paths

- `apps/web/src/pages/{Machines,Machine,Pair,Schedules,Plugins,Settings,History,Usage}.tsx`, `apps/web/src/pages/{machines,ops}/**`, `apps/web/src/router.ts` (`/history`, `/usage` real pages), `apps/web/src/styles/pages.css` (own sections only), `apps/web/__tests__/pages/ops*.test.tsx`, `apps/web/__tests__/routes.test.ts`, `apps/web/e2e/ops.spec.ts`, `docs/architecture.md` §10 (routes list)

## Depends on

#85, #87

## Scope

- **Machines**: one bordered group per machine (glyph 20, name, OS + heartbeat age, status pill), environments in three columns (`repeat(3, minmax(0, 1fr))`, gap 12) of `EnvironmentCard`s; an offline machine with queued work states "N tasks queued until it returns · will not move to another account" (EXE-12); the `platform` row at the bottom represents `anthropic-api`. No machines → `platform` row + the dashed "Pair a machine" empty state. Topbar action `Pair a machine` → `/pair`.
- **Machine**: header glyph + name 24/600 + status; grid `1fr 420px`. Left: environment cards, active sessions `DataTable` (session id link, agent tile, environment, status, age) — export this table component for the agent Sessions tab. Right: Doctor checklist for EXE-07 (`Timeline`/list with check or cross per item), Revoke in an `error`-bordered card whose `ConfirmDialog` says sessions become `disconnected`, not failed.
- **Pair**: max-width 620 + 360 rail. Three numbered `Steps`; the code as six 64×80 cells, mono 40/600 in primary; countdown from 10:00 (zero `Countdown`); at 0 the cells go text-dim and a `New code` button replaces the waiting line; on success step 2 gets the check and the page links to the new machine.
- **Schedules**: `DataTable` `110px 1fr 140px 140px 310px 44px`: kind `Tag`, what (+ amber policy line when bound to an offline environment), when, next run, runs on (environment line or agent tile), enable `Switch`. Topbar `New schedule`. Footer line states the DST rule (AST-07).
- **Plugins**: three-column cards: name + version, kind `Tag`, description, granted permissions, declared unsupported operations (PLG-09), dependents as tiles, enable `Switch`. Turning a switch off with dependents opens the `ConfirmDialog` drawn on the board ("Disable and stop 2 sessions").
- **Settings**: max-width 860, two-column form sections (240 px title column): time zone (`SettingsForm` from #73), notifications matrix (event × inbox, push — hide the push column when push is unavailable), API keys (masked value, status pill, `Replace`), budgets, retention, export and delete (danger).
- **History** `/history`: filter chips by kind (`aria-pressed`); `DataTable` `84px 150px 130px 1fr 110px`: time (mono), kind `Tag`, actor (tile + name), what happened, ref link. Newest first, grouped by day with a day heading row.
- **Usage** `/usage`: `Segmented` by agent / task / turn; four stat cards (`repeat(4, minmax(0, 1fr))`, mono 28/600); cost-per-day bars; `DataTable` `1fr 120px 120px 130px 150px` with the mandatory data-quality column (`REPORTED`, `PARTLY ESTIMATED`, `NOT REPORTED`); unreported cost prints `n/a` in text-dim, never 0 or blank; estimates are prefixed `~` (OPS-07).
- `router.ts` + routes test: `/history`, `/usage` become real pages; `docs/architecture.md` §10 lists them and `/tasks`.
- `pages.css` sections per page.

## Out of scope

- Live Machine / Schedule / Registry / Audit / Ledger reads and mutations (#36, #42, #48, #44, #45); mobile.

## Acceptance

- [ ] happy-dom tests per page: Machines renders the platform row and the EXE-12 line for the offline machine; Pair renders six code cells and swaps to `New code` when the countdown model hits 0; Schedules shows the policy line for the offline-bound row and the DST footer; Plugins opens the dependents dialog on disable; Settings hides the push column when push is off; History groups by day newest first; Usage prints `n/a` + `NOT REPORTED` for unreported cost and never `0`.
- [ ] Routes test covers `/history` and `/usage`; e2e at 1280 visits all eight routes with no console errors and asserts the table column templates.

## Verify

```sh
pnpm typecheck && pnpm lint && pnpm test apps/web && pnpm --filter @agentic/web build
pnpm --filter @agentic/web test:e2e
```

## Read first

`docs/design/HANDOFF.md` → Screen specs, Layout and shell (tables), Edge cases; `packages/ui/src/kit/**`, `packages/ui/src/forms/{environment-card,settings-form}.tsx`; `docs/architecture.md` §4 Machine / Schedule / Registry / Audit / Ledger, §10. Then `AGENTS.md` → "Picking up an issue". Part of the tracking issue #11.

### PR DESCRIPTION
**Pull request by andtii**
Closes #90.

The eight operations routes on mock data, laid out to `docs/design/HANDOFF.md` (Machines through Usage) with the kit (#99) and the states (#101). Each page is a `*View` component over props plus a thin route component that reads `apps/web/src/mock/ops.ts`, so #36, #42, #48, #44 and #45 swap the data source without touching the layout.

**Pages**
- `Machines` — one bordered group per machine (glyph, name, OS · daemon · heartbeat, status pill, Details), its `EnvironmentCard`s in `repeat(3, minmax(0, 1fr))`, the EXE-12 line under an offline machine with queued work ("will not move to another account or machine by itself"), the `platform` row for `anthropic-api` at the bottom, and the dashed `EmptyState variant="machines"` when there is no machine.
- `Machine` — header, environments "reported by the daemon", the active sessions `DataTable` (`110px 1fr 140px 140px 120px`, exported as `pages/machines/SessionsTable` for the agent Sessions tab), the EXE-07 doctor checklist, and the revoke card whose `ConfirmDialog` says sessions become disconnected, not failed ("Revoke and disconnect 2 sessions").
- `Pair` — hero title, three numbered steps, command wells with Copy, six 64 × 80 code cells in mono 40 / 600 primary, a countdown (`Countdown`) from what is left of 10:00; at zero the cells go dim and `New code` replaces the waiting line. The rail explains what pairing grants.
- `Schedules` — `DataTable` `110px 1fr 140px 140px 310px 44px`: kind `Tag`, what (+ the amber `nuc-lab is offline · policy: queue until it returns` line under the offline-bound row), when, next run (`paused` while disabled), runs on (platform text, or agent tile + `EnvironmentLine`), enable `Switch`. Footer states the DST rule (AST-07).
- `Plugins` — three-column cards: name + version, kind tag, description, GRANTED list, DECLARED UNSUPPORTED (PLG-09), "Used by" tiles or "No dependents", Configure, enable switch. Switching off a plugin with dependents opens the `ConfirmDialog` listing them by name and stating the consequence; the switch holds until confirmed.
- `Settings` — max-width 860, two-column sections (240 px title + hint): time zone and default environment, the notifications matrix (event × Inbox, Push — the Push column is hidden, not disabled, when push is unavailable), API keys (masked, `KEY OK` pill, Replace), budgets, retention, export and the delete confirm.
- `History` (`/history`, real page now) — filter chips with `aria-pressed`, `DataTable` `84px 150px 130px 1fr 110px`, newest first and grouped by day with a day heading row; kind tones: approval asked amber, interrupted red, correction live.
- `Usage` (`/usage`, real page now) — `Segmented` by agent / task / turn, four stat cards, cost-per-day bars, `DataTable` `1fr 120px 120px 130px 150px` with the mandatory data-quality column. Unreported cost prints `n/a` in text-dim, never 0; estimates are prefixed `~` and carry `PARTLY ESTIMATED` (OPS-07).

**Acceptance**
- [x] happy-dom tests per page (`apps/web/__tests__/pages/ops.test.tsx`, 18 cases): Machines renders the platform row and the EXE-12 line for the offline machine only, and the empty state with no machines; Machine renders the sessions template, the doctor list and the revoke dialog wording; Pair renders six cells and at `expiresIn={0}` swaps to `New code`, which restarts at 10:00; Schedules shows the policy line on the offline-bound row only, `paused`, and the DST footer; Plugins opens the dependents dialog on disable (all three dependents named, "Disable and stop 2 sessions", switch held until confirm) and needs no dialog without dependents; Settings hides the push column when push is off and every field has a label; History groups by day newest first and filters with pressed chips; Usage prints `n/a` + `NOT REPORTED`, `~$6.52` + `PARTLY ESTIMATED`, and no cost cell is `0`.
- [x] `routes.test.ts` covers `/history` and `/usage` as real pages; e2e `ops.spec.ts` at 1280 visits all eight routes with no console error, asserts the four table column templates, no horizontal scroll, the Machines → Machine and → Pair links with the countdown ticking, and the Plugins dialog. Ran locally: 10 passed (mobile project skipped, #91).
- [x] `pnpm typecheck && pnpm lint && pnpm test && pnpm size` and `pnpm build` green; `zero:validate --strict` 0 errors, 0 warnings. Size unchanged (`@agentic/ui` 62.73 kB of 66, css 20.16 kB of 22): nothing in `packages/ui` changed.
- [x] `docs/architecture.md` §10 gains the ops-pages line (routes were already listed by #94).

**Decisions**
- Page actions live in a head row inside the content, right-aligned, not in the topbar the artboards draw. The topbar's `actions` slot is wired in `App.tsx`, which is outside this issue's owner paths; a `usePageActions` seam there is a small follow-up (noted for #91 / the shell).
- `pages/ops/OpsPage` replaces `components/Page` for these routes: the breadcrumb is the title on every ops page, so the `h1[data-page-title]` is visually hidden (Pair shows it as the hero). `components/**` is #88's, so nothing there changed.
- `pages/ops/LinkButton` is a real `<a href>` stamped with the button anatomy and pushed through the router on click; `pages.css` sets `color: revert-layer` on it because the document's unlayered `a { color }` beats the layered button recipe.
- Rows that carry `data-*` (`data-schedule`, `data-history-row`, `data-usage-row`, the day heading) are raw `<tr data-scope="table" data-part="row">`: zero's `Table.Row` forwards only `class` and `selected`, and `Table.Cell` takes no `colspan`.
- Settings composes the kit's `SelectField` / `TextField` and its own matrix rather than `SettingsForm`, whose one push switch is not the artboard's event × channel matrix. The form's model helpers stay the persistence path for #35 / #14.
- `mock/data.ts` is untouched (#88 owns it); ops data lives in `mock/ops.ts` on core's `EnvironmentDescriptor` / `MachineInfo` / `NotificationKind`.
- The Pair grid is `minmax(0, 620px) minmax(280px, 360px)` so 1280 fits (the drawn 620 + 360 overflowed the 992 px content column); the mobile pass (#91) handles < 768.
- Renewing an expired code reuses the sample code until #36 supplies a real one.

**Not changed**: `packages/ui`, `App.tsx`, `nav.ts`. No zero-wip issue needed.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: apps/web/src/pages/Pair.tsx [hunk 1]
```diff
@@ -1,26 +1,121 @@
...
+    };
+    onMounted(start);
+    onUnmounted(stop);
+    // A new code from the platform lands with #36; until then the same sample code restarts the clock.
+    const renew = () => {
+        state.code = props.code;
+        state.remaining = CODE_LIFETIME;
+        start();
+    };
+    const copy = (text: string) => { void navigator.clipboard?.writeText(text); };
```
[the remark this answers is not in the document]
Comment (andtii) [reply apps/web/src/pages/Pair.tsx]: Fixed in 72baeed: the write now ends in `.catch(() => {})` — a refused clipboard is not an error the page reports.

File: apps/web/src/pages/ops/format.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+export function dayLabel(iso: string): string {
+    return DAY.format(new Date(iso));
+}
+
+/** "14:20:03" — the time part of an ISO local time. */
+export function clock(iso: string): string {
+    return iso.slice(11, 19);
+}
+
+/** Group ISO-timed items by calendar day, newest day first; items keep the order given. */
```
[the remark this answers is not in the document]
Comment (andtii) [reply apps/web/src/pages/ops/format.ts]: Fixed in 72baeed: the comment now says newest first for the days and for the items inside each day, which is what the sort does and what the History test asserts.
